install/image-references

```diff
@@ -27,6 +27,10 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-pod:v4.0
+  - name: etcd-quorum-guard-image
+    from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/openshift/origin-v4.0:base
```

Comment:
this should reflect this to be correct https://github.com/openshift/etcd-quorum-guard/blob/master/manifests/image-references#L5-L8 right?

Reply:
https://github.com/openshift/etcd-quorum-guard/ is actually now dead code (at least until/unless we revive it as a full-fledged operator).